conf/prod.routes

```diff
@@ -1,5 +1,13 @@
 # Add all the application routes to the app.routes file
-->         /single-customer-account-stub  app.Routes
```

Comment:
removing app.routes might be problematic, if another service has the same endpoint in one of the environments it would clash

Reply:
the point here is, we should have same endpoint except host+port. By doing this, we can override the environment variables for each external service(api), we can achieve real(qa,prod) or stub data (local, dev, staging).